### ISSUE TITLE
Robot 1 doesn't load OS

### ISSUE THREAD

**Issue by mmetcalfe** [NUbots/robocup#11]
Could not SSH into the robot.
Connected a monitor via the HDMI port to see what was happening.
Was presented with the GRUB rescue prompt:

GRUB loading.
error: out of partition
grub rescue>

**Comment by BrendanAnnable** [comment]
This happened after receiving a read-only file system error and then not loading after the reboot.

No idea what the cause was.

**Comment by josiahw** [comment]
I have a feeling this was linked to the screw smashing into the motherboard after it came back from repair. We should try re-imaging the os. Maybe even use it to test a new distro. If that fails, we send it to Peter.

**Comment by BrendanAnnable** [comment]
Using it as a test robot for a new OS - see #7

**Comment by BrendanAnnable** [comment]
99.99% sure this robot has hardware issues.

Attempted to load Ubuntu in a variety of ways (alternate, minimal, server, etc).

Received many booting/partition/IO errors. I ran a diagnostic utility which found 4000 bad sectors and swapped them out, which I hoped would fix it. Unfortunately it didn't and continued to have problems when installing Ubuntu. Found another 4000 bad sectors today and seemed to failing to remember anything stored on the drive.

My conclusion is that the NANDrive is broke. It seems to have lost its ability to store information and is now volatile.

A suggestion was made to use the SD card slot as a replaceable HDD and install the OS on it, it would make re-imaging quite easy as we could just remove the SD card and re-image and slot it in.

For the mean time it looks like we have to switch the main motherboard, I'll happily give it a go if you're okay with that @josiahw?
